### ISSUE TITLE
Chrome + NVDA heading within summary negates announcement of state change

### ISSUE THREAD

**Issue by scottaohara** [nvaccess/nvda#8706]
Using the latest Chrome, interacting with a `details` and `summary` pattern behaves differently if a heading is a child of a `summary` element.

### Steps to reproduce:
* With NVDA and Chrome, visit: http://www.bleachedgraphics.com/test/details-summary/
* Navigate to Trigger 1 and expand/collapse the trigger to hear the announcements
* Navigate to Trigger 2 and expand/collapse the trigger to hear the announcements

### Actual behavior:
* Trigger 1 announces "expanded" and "collapsed" when pressing <kbd>space</kbd> or <kbd>enter</kbd> to toggle the disclosure widget.
* Trigger 2 will make no announcement of "expanded" or "collapsed" when performing the same actions.

### Expected behavior:
Regardless of whether a `summary` contains a heading or not, NVDA should announce the updated state of the disclosure widget when toggled with <kbd>space</kbd> or <kbd>enter</kbd>

#### NVDA Installed/portable/running from source:
installed

#### NVDA version:
2018.2.1 

#### Windows version:
Windows 10 Pro
10.0.17134

#### Name and version of other software in use when reproducing the issue:
Chrome (68.0.3440.106) 64 bit

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA?
No

**Comment by leonardder** [comment]
cc @michaeldcurran @jcsteh

**Comment by scottaohara** [comment]
Adding onto this issue, testing again today with NVDA 2018.4.1 the original issue still exists.

Additionally, testing with Firefox 64.0.2, no state change is announced at all when toggling a details/summary widget to be expanded or collapsed.

**Comment by Adriani90** [comment]
This issue still exists in NVDA 2019.3 Beta 1 and Google Chrome 79.
I think it is related to the offsets of the elements and how they are registeres to the virtual buffer. In Chrome the virtual cursor cannot detect exactly the system focus and in browse mode the system focus cannot reliably be drawn to the heading. This seems to work in Firefox much more reliable.
If you turn off automatically move system focus to focusable elements (nvda+8), it will work in Chrome just properly.
cc: @leonardder

**Comment by Adriani90** [comment]
cc: @ObjectInSpace

**Comment by ObjectInSpace** [comment]
As it seems this does not reproduce in JAWS or Firefox, I Filed https://crbug.com/1098060 to track the issue. Thanks for reporting it!

**Comment by Adriani90** [comment]
This seems fixed since Chrome 93 or earlier. Closing as works for me.
